inst/scripts/SEL3D.R

```diff
@@ -0,0 +1,139 @@
+#Pseudocode may look like actual code...please ignore...
+getPdbIds <- function(targetSeq, startPos = 1 , endPos = nchar(targetSeq), convert = TRUE, silent = TRUE){
```

Comment:
I'm not sure if you can have endPos depend on targetSeq this way. Another way to do this is the way Boris did it last class.
if(missing(endPos)) {
    endPos <- nchar(targetSeq)
}